operations/iscsi_sbps/iscsi_sbps.md

```diff
@@ -31,6 +32,9 @@ In CSM 1.6, both DVS and SBPS will co-exist and DVS is planned to deprecate in C
 * Aligns with future plans for similar functionality in next generation systems management solutions
 * Easy to deploy and manage
 
+**Note:** Using HSN for boot content projection is recommended and use NMN for any debugging purposes.
```

Comment:
Added a comma.

**Note:** Using HSN for boot content projection is recommended, and use NMN for any debugging purposes.

Reply:
Done